GitUI/Editor/FileViewerInternal.cs

```diff
@@ -15,6 +16,8 @@ public partial class FileViewerInternal : GitExtensionsControl, IFileViewer
         private DiffHighlightService _diffHighlightService = DiffHighlightService.Instance;
         private bool _isGotoLineUIApplicable = true;
 
+        public Action OpenWithDifftoolAction { get; private set; }
```

Comment:
I don't think the suffix adds value.  Suggest `public Action OpenWithDifftool { get; private set; }`.
Same for `IFileViewer.cs`